Error: A `label` alone cannot be used to provide an accessible label for a group of fields. Labels can only be associated with a single field using the `for` attribute. Use `fieldset` and `legend`. Error: Checkboxes in a group should use a `name` value that ends with "[]" to allow server-side code to collect the values as an array (or similar). Note: If the checkbox control is not nested in its `label`, wrapping the two elements within another element (such as a `div`) is preferable. Having a shared parent element gives designers more control over the eventual layout of the checkbox group than a `br` does.
diff --git a/output/Checkbox Group/1/eedcb723-a9dc-4d0e-9b50-92f352909de3.html b/output/Checkbox Group/1/eedcb723-a9dc-4d0e-9b50-92f352909de3.html
index ee00983..06dfeec 100644
--- a/output/Checkbox Group/1/eedcb723-a9dc-4d0e-9b50-92f352909de3.html	
+++ b/output/Checkbox Group/1/eedcb723-a9dc-4d0e-9b50-92f352909de3.html	
@@ -1,7 +1,15 @@
-<label>What fruits do you like?</label><br>
-<input type="checkbox" id="bananas" name="fruits" value="bananas">
-<label for="bananas">Bananas</label><br>
-<input type="checkbox" id="oranges" name="fruits" value="oranges">
-<label for="oranges">Oranges</label><br>
-<input type="checkbox" id="apples" name="fruits" value="apples">
-<label for="apples">Apples</label>
\ No newline at end of file
+<fieldset>
+  <legend>What fruits do you like?</legend>
+  <div>
+    <input type="checkbox" id="bananas" name="fruits[]" value="bananas">
+    <label for="bananas">Bananas</label>
+  </div>
+  <div>
+    <input type="checkbox" id="oranges" name="fruits[]" value="oranges">
+    <label for="oranges">Oranges</label>
+  </div>
+  <div>
+    <input type="checkbox" id="apples" name="fruits[]" value="apples">
+    <label for="apples">Apples</label>
+  </div>
+</fieldset>
\ No newline at end of file
